### ISSUE TITLE
Plans for making 20.04 since 19.04 is now unsupported?

### ISSUE THREAD

**Issue by billyblackburn** [yusefnapora/pixelbook-linux#59]
I read that 19.04 is now unsupported so are there any plans or anyone able to run 20.04 and get it working?

**Comment by kyzn** [comment]
I tried 20.04, but with the current 4.4.205chromium kernel (at [flantel/pixelbook-linux](https://github.com/flantel/pixelbook-linux)), I got stuck at a blank screen.
I wonder if it would be easy to set up to use a newer kernel from https://chromium.googlesource.com/chromiumos/third_party/kernel/

Also for reference: https://en.wikipedia.org/wiki/Ubuntu_version_history

**Comment by kyzn** [comment]
Ok, looking at https://github.com/flantel/pixelbook-linux/blob/master/implementation-details.md and https://github.com/megabytefisher/eve-linux-hacks now...

**Comment by kyzn** [comment]
eve kernel looks like `4.4` here, so I'll try to get latest 4.4.
https://www.chromium.org/chromium-os/developer-information-for-chrome-os-devices

**Comment by kyzn** [comment]
Well, I couldn't make it work. But make sure to look at other issues:
- Going back to 18.04 LTS: https://github.com/yusefnapora/pixelbook-linux/issues/58
- 20.04: https://github.com/yusefnapora/pixelbook-linux/issues/50
- Kernel 5.4: https://github.com/yusefnapora/pixelbook-linux/issues/44

**Comment by billyblackburn** [comment]
@kyzn ok thank you for trying, maybe we can get there eventually. or make

**Comment by Ikbosh** [comment]
@billyblackburn @kyzn I've made a comment on the other issue about 20.04, I have got 4.4 working on MX Linux 19.2 (Debian Buster base) I wonder if that would be enough for you to get 20.04 working? https://github.com/yusefnapora/pixelbook-linux/issues/50#issuecomment-693342925

**Comment by kyzn** [comment]
Hi @Ikbosh , thanks for your detailed response! Unfortunately I went back to Chrome OS a while ago. If I come back around I will make sure to give it a go.

**Comment by allezacc** [comment]
so any update on this?

**Comment by ackstorm23** [comment]
> so any update on this?
> 

Pretty sure the developer of this cookbook no longer maintains it. 

In fact many of the people who appear to hve been active in the past have stopped using their pixelbook or gone back to chromeos. 

I am running 20.x but in order to fix the display issue I am running the stock Ubuntu kernel

It works, just no sound or backlight control.

Someone else did get a newer version of the 4.4 chromium kernel running but did not submit any patches to this cookbook. 

Until more people from the community find time to contribute, the less technical users will be stuck.

**Comment by allezacc** [comment]
I would like to help, but I have no idea on how I could make this work

**Comment by ackstorm23** [comment]
> I would like to help, but I have no idea on how I could make this work

most of the ansible code still works fine.

the part that is broken is the version of the 4.4 chromium kernel that the maintainer installs.  

there are newer versions of 4.4 available and changing the ansible configuration to grab and use the newer version name is fairly simple once you understand what he is doing.

changing some variable names to the new kernel version,  a url path to find it, and providing a kernel config file relevant to that kernel. 

ansible will do the compiling for you, as long as you pass the parameter to ansible to build from source instead of prebuilt package (default).

**Comment by Elara6331** [comment]
I have found that if you install the whole thing on any distro newer than 19.10, it gets stuck on a black screen. This appears to be caused by systemd as I tried artix with runit and it worked flawlessly.

**Comment by Ikbosh** [comment]
> 
> I am running 20.x but in order to fix the display issue I am running the stock Ubuntu kernel
> 
> It works, just no sound or backlight control.
> 
> Someone else did get a newer version of the 4.4 chromium kernel running but did not submit any patches to this cookbook. 
> 

If you're referring to me, in the other thread about 20.X I mentioned I posted all my changes to my fork of the repo (I'm not good at Git). I too sadly have changed back to ChromeOS at the moment due to buggy USB C dock support on Linux Kernel 4.4, I'll likely go back when I have more time again to troubleshoot and workout what modules / config changes I need to make to get it working the way I desire.

**Comment by ackstorm23** [comment]
non-chromium kernels work fine,  but backlight and audio don't work. 

there is a patch for backlight, although it has not been updated for newer kernels.

I have not found a patch extracted for audio though, and that is probably what matters the most to users. 

I've seen people on other chromebooks do modifications to module options with snd-hda-intel and/or the snd-soc-skl driver, but in my testing thus far I have not gotten it working.
